lighthouse-core/gather/computed/manifest-values.js

```diff
@@ -67,7 +67,7 @@ class ManifestValues extends ComputedArtifact {
       {
         id: 'shortNameLength',
         failureText: 'Manifest `short_name` will be truncated when displayed on the homescreen',
-        validate: manifest => manifest.value.short_name.value &&
+        validate: manifest => !!manifest.value.short_name.value &&
```

Comment:
added the `!!` so that it's always a boolean. If the site had no `short_name`, this would return `undefined` which would then get `JSON.stringified` out of existence in the JSON report